Src/DependencyCollector/Shared/Implementation/DependencyCollectorEventSource.cs

```diff
@@ -266,9 +266,17 @@ private string GetApplicationName()
             string name;
             try
             {
-#if NETCORE
+#if NETSTANDARD1_3
+                // Assembly.GetEntryAssembly() is not available on netstandard1.3, but is available in runtime
+                var getEntryAssemblyMethod =
+                    typeof(Assembly).GetMethod("GetEntryAssembly", BindingFlags.Static | BindingFlags.NonPublic) ??
+                    typeof(Assembly).GetMethod("GetEntryAssembly", BindingFlags.Static | BindingFlags.Public);
```

Comment:
Why not combine the BindingFlags for Public and NonPublic into a single call instead of doing the ?? check?